### PR TITLE
feature/update-gradle feat: Update build gradle

### PR DESCRIPTION
**Pull request by juancoob**
**SUMARY**

Bring up to date the gradle file.

**CHANGES MADE**

On the project's build.gradle, I have added the last gradle version.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.1.2"
+        classpath 'com.android.tools.build:gradle:4.1.3'
```
Comment (jujoru) [inline build.gradle:9]: This is a comment

**Review by joseantoniocampillo — APPROVED** [review]
That' s ok

File: build.gradle (hunk 1 above)
Comment (juancoob) [reply build.gradle:9]: Perfect

**Review by juancoob — COMMENTED** [review]
Answer @jujoru

**Comment by jujoru** [comment]
Perfect

File: build.gradle (hunk 1 above)
Comment (JorgeGreek) [inline build.gradle:9]: comment Jorge.

**Review by JorgeGreek — APPROVED** [review]
perfect

**Review by jujoru — APPROVED** [review]
Aprove ¡¡¡

**Review by jujoru — APPROVED** [review]
Other Approve

**Review by JorgeGreek — APPROVED** [review]

**Review by JorgeGreek — APPROVED** [review]

**Review by joseantoniocampillo — APPROVED** [review]
That's all right

**Comment by juancoob** [comment]
Update gradle version.

**Pull request merged**